changelog/7498.feature.md

```diff
@@ -0,0 +1,14 @@
+Incremental training of models in a pipeline is now supported.
```

Comment:
don't we an issue for incremental training? In that case we should use the issue number in the changelog name.